### PR TITLE
Feat/results page/chl-1897

### PR DESCRIPTION
**Pull request by rohan-1898**

### CODE REVIEW

**Review by shanu798 — APPROVED** [review]
review done

File: src/features/results/result-page.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      margin-top: 1.6rem;
+    }
+  }
+
+  &__card{
+    &-container{
+      border: none;
+      margin-top: 5.8rem;
+      display: flex;
+      flex-direction: row;
```
Comment (vaibhavg15) [inline src/features/results/result-page.scss:28]: you don't need to add flex-direction: row. Default is row only

**Review by vaibhavg15 — CHANGES_REQUESTED** [review]

File: src/features/results/result-page.scss (hunk 1 above)
Comment (rohan-1898) [reply src/features/results/result-page.scss:28]: in the cardBase, from ui-kit flex-direction is set to column 
so, I had to update

**Review by vaibhavg15 — APPROVED** [review]
peer review done

File: src/features/results/ResultsPage.tsx [hunk 2]
```diff
@@ -1,3 +1,103 @@
...
+        size={SIZE.M}
+        className={styles.header.description}
+      >
+        {RESULTS_PAGE_STATICS.HEADER.description}
+      </TextBody>
+    </div>
+  );
+
+  const getCardView = (cardData:any) => {
+    const noteCustomClassName = classNames({
```
[the remark this answers is not in the document]
Comment (rohan-1898) [reply src/features/results/ResultsPage.tsx:59]: added dataType to cardData

**Pull request merged**